tests/main/special-home-can-run-classic-snaps/task.yaml

```diff
@@ -18,8 +18,8 @@ prepare: |
             wget -q -O - https://pkg.jenkins.io/debian/jenkins.io.key | apt-key add -
             echo 'deb http://pkg.jenkins.io/debian-stable binary/' > /etc/apt/sources.list.d/jenkins.list
             apt-get update
-            apt-get install -y jenkins
-            apt install jenkins
+            # XXX: remove --allow-unauthenticated once jenkins repo is updated.
+            apt-get install -y --allow-unauthenticated jenkins
```

Comment:
I'd rather see us disable this test for now and open a concurrent PR re-enabling this test and just re-start the test everyday until the upstream issue is fixed.